framework/generated/generated_dx12_replay_consumer.cpp

```diff
@@ -5719,24 +5719,26 @@ void Dx12ReplayConsumer::Process_ID3D12Device4_CreateCommittedResource1(
     Decoded_GUID                                riidResource,
     HandlePointerDecoder<void*>*                ppvResource)
 {
-    auto replay_object = MapObject<ID3D12Device4>(object_id);
-    if (replay_object != nullptr)
+    auto replay_object = GetObjectInfo(object_id);
+    if ((replay_object != nullptr) && (replay_object->object != nullptr))
     {
```

Comment:
I'm pretty hazy on how the generated code changed so much as a result. @davidd-lunarg If you get any time, your eyes would be useful on this.

Reply:
This looks correct to me. Before this change, the generated code was resolving the objects to their native DX12 types and making the DX12 API call to `CreateCommittedResource1/2`. After this change, the objects are resolved to GFXR types (e.g., `DxObjectInfo`) and passed to `Dx12ReplayConsumerBase::OverrideCreateCommittedResource1/2` which is responsible for resolving them to the native DX12 types and making the actual DX12 API call to `CreateCommittedResource1/2`.